be/src/column/column_hash.h

```diff
@@ -314,8 +314,8 @@ inline void hash_combine(uint64_t& seed, const T& val) {
 }
 
 inline uint64_t hash_128(uint64_t seed, int128_t val) {
-    size_t low = val;
-    size_t high = val >> 64;
+    auto low = static_cast<size_t>(val);
```

Comment:
size_t?

Reply:
If the right side of the equal sign is static_cast, the type of the left table is very intuitive, there is no need to write the type, the IDE will give a prompt